vector/src/main/java/im/vector/app/features/home/room/detail/composer/MessageComposerViewModel.kt

```diff
@@ -204,134 +204,134 @@ class MessageComposerViewModel @AssistedInject constructor(
             when (state.sendMode) {
                 is SendMode.Regular -> {
                     when (val slashCommandResult = commandParser.parseSlashCommand(action.text)) {
-                        is ParsedCommand.ErrorNotACommand         -> {
+                        is ParsedCommand.ErrorNotACommand, ParsedCommand.ErrorNotATchapCommand -> {
```

Comment:
Please, can you move the second command to a new line? It will probably avoid reformating the other lines